sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatClientTestBase.java

```diff
@@ -45,7 +45,7 @@
     protected static final String CONNSTRING = Configuration.getGlobalConfiguration()
         .get("COMMUNICATION_SERVICES_CONNECTION_STRING", "pw==");
 
-    protected static final FixedDelay RETRY_STRATEGY = new FixedDelay(3, Duration.ofMillis(500));
+    protected static final FixedDelay RETRY_STRATEGY = new FixedDelay(5, Duration.ofSeconds(1000));
```

Comment:
Maybe we should disable the tests that have this issue and fix the underlying problem.

Reply:
We can disable the tests, but unfortunately the underlying problem is "by design" according to Chi's findings